cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -66,6 +67,9 @@ elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "ghost"* ]] \
   || [[ $ATDM_CONFIG_REAL_HOSTNAME =~ gho[0-9]+ ]] ; then
   ATDM_HOSTNAME=ghost
   ATDM_SYSTEM_NAME=serrano
+elif [[ $ATDM_CONFIG_REAL_HOSTNAME == "doom"* ]] ; then
```

Comment:
@fryeguy52, FYI: After the merge of PR #6231, you will need to merge 'develop' into this PR branch or rebase this PR branch on top of 'develop' and add back this elif statement.  It should be clear how to do that if you look at the new file.